src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+    private List<Integer> numbers;
```

Comment:
```suggestion
    private Set<LottoNumber> numbers;
```
- 로또 번호는 중복이 될 수 없습니다. 그래서 List 대신에 Set으로 표현해주면 어떨까요?
- 숫자 정렬을 위해 TreeSet으로 구현해주면 어떨까요?
  - Set은 interface고, TreeSet은 implement 입니다!
- Integer를 사용하기보단, 1 ~ 45의 숫자만 가질 수 있는 LottoNumber 라는 원시값 포장 객체를 만들어주면 어떨까요?

Reply:
```java
Set<LottoNumber> = new TreeSet<>();
``` 
이렇게 사용하니 훨씬 깔끔하고 sort가 따로 필요 없어서 더 좋네요! 
> 알고리즘으로 해결하기보단 자료구조로 해결하기(ex: 중복처리 ->Set, 정렬 -> TreeSet) 

1단계 문자열계산기에서 이렇게 언급해주셨는데 놓친 부분이네요...! ㅠㅠ 참고해서 수정했습니다!ㅎㅎ